core/controllers/story_editor.py

```diff
@@ -59,6 +60,49 @@ class EditableStoryDataHandler(base.BaseHandler):
     """A data handler for stories which support writing."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'story_id': {
+            'schema': {
+                'type': 'basestring'
+            },
+            'validators': [{
+                'id': 'has_length',
+                'value': constants.STORY_ID_LENGTH
+            }]
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'version': {
+                'schema': {
+                    'type': 'int',
+                    'default_value': None
+                }
+            },
+            'commit_message': {
+                'schema': {
+                    'type': 'basestring',
+                    'default_value': None,
```

Comment:
Why can this be `None`?

Reply:
I was under the impression that commit messages for making changes were optional and wanted to check for that case. Should I remove it?

Reply:
Not sure, you will need to go through the codebase and see if we always pass some values.

Reply:
Ah my bad, after checking, it looks like commit messages can't be `None` by default.